### ISSUE TITLE
A few possible bugs.

### ISSUE THREAD

**Issue by ccfox** [settintotrieste/Victoria-2-Cold-War-Enhancement-Mod-CWE#810]
I found a few issues coming back to this after so many years, my apologies that I likely won't be able to help to the degree I had before.

- Algeria should probably be able to switch its culture from French as long as it isn't a puppet of France.

- Arab countries should probably be able to embrace pan-Arab ideals even if a puppet if they are puppeted by the UAR. I also think that communist nations should be able to form the UAR, since they can already embrace pan-Arabism, though that is a minor thing and not really a bug.

- Hawai'i still seems to barely have many Polynesian pops, looking at demographics from that time it should be just a little under the Anglo-American population.

- This may have been fixed in the most recent round of fixes, but mentioning it since I haven't gotten a chance to test. If the cold war ends in the favor of the USSR, East Germany is annexed by FRG as a western nation rather than a communist-aligned state.

- The localization for the dispute between Turkey and the USSR seems to be broken for whatever reason.

- Iraq is able to make a very weird flex and puppet an independent Kurdistan. 

Also, some suggestions:

- I vaguely remember writing an event for Danzig ages ago. I'd like to put forward again that it be added since currently the USSR doesn't really have any recourse, whereas with the event it would get the option to undergo the normal process of the rest of the countries currently.

- I noticed that Eurasia doesn't get many accepted cultures. I thought it would be neat rather than just adding more cultures to the list to make an event modeled after the Ottoman reform event in PDM, where a player that forms Eurasia can choose between aligning more with Asia and earning Central Turkic and potentially Mongol/Manchu accepted cultures, Europe with a focus on Eastern European cultures, or a balance that adds a bit of both. 

- Communist Greece, if it somehow manages to attain victory, should become a Soviet Puppet (and likely tag-switch to Greece if Greece doesn't still exist). 

- The Soviets being able to successfully support the Kurds and Azeris in Iran should instantly wp and annex Azerbaijan and make Mahabad a puppet. 

- One of the options for the outcome of protests against big C in Romania should be a Socialist Democracy with Market Socialists at the helm (or a Presidential Dictatorship with Market Socialists if you think its more fitting to the weird Stalinist Reformism of Apostol) rather than Western-style Democratization (One of the main opposition figures within the Romanian Workers' Party, Apostol, was freed by the protesters and was at least somewhat popular and iirc a lot of figures in the party that hated Ceaușescu participated in the movement against him). Really it would be neat to see the entire 'fall' revised to allow it to be more dynamic and allow more options and ways to stop or stave it off - or for other large communist nations to intervene. 

For the 1992 start date, so feel free to ignore if you don't feel like dealing with it:

- The 'Republic of Afghanistan' was technically still around until April of 1992 as a multi-party democracy (dominated by a single party; the Parcham faction of the PDPA that had abandoned hard-line Marxism Leninism just two years prior and renamed itself to 'Watan Party'), though on the brink of collapse. It would likely take a lot of effort for the 92 start unless you made the final push to Kabul an event that allowed the player (or ai, I suppose) to choose the alt-history path of it being averted in exchange for one of the existing malus modifiers (though obviously a more 'accurate' portrayal would be insanely complicated).

- Mongolia's ruling party should be the socialist MPRP (I'd completely remove the 'Social Democratic Party' and rename the MPRP's socialist faction to something like 'MPRP - Liberals' or something).

**Comment by ccfox** [comment]
Also as a bonus here is a - probably poorly coded - event for the Mongolian Constitution since despite its democratization protests happening in 1990 was not written until 1992 and seemed to be written by the MPRP itself (technically the Mongolian People's Republic' also wasn't abolished until then, though effectively they had reformed into a multi-party state already).
[Mongolia 1992.txt](https://github.com/settintotrieste/Victoria-2-Cold-War-Enhancement-Mod-CWE/files/6740708/Mongolia.1992.txt)

**Comment by settintotrieste** [comment]
Hi Chrisfox, massive thanks for the feedback and suggestions! I've looked at them and done the changes save those mentioned below:

6. The weird flex is due to a necessary event to establish the Autonomous Kurdistan state in Iraq as there is no other good way I can think of to do it.

7. I'm afraid I can't remember what was the proposed Danzig event and don't have a copy of it anymore.

8. Vicky 3 is coming soon so I'm gonna give it a pass. Creating any superstate add/remove culture system for this mod is quite labour intensive due to a strange hardcoded bug that makes any subsequent changes to the cultures of the current country tag also change those of the previous country tag, if change_tag was used. This means an additional system is needed to track which country the superstate was formed from.

11. I dropped the communist intervention part as it was too complex to script.
